### DISCUSSION TITLE
Examples of where Layout grid can be used

### DISCUSSION THREAD

**Discussion by gitcode7** [uswds/uswds#4876] [General]
Hello,

I was trying to use [USWDS Layout grid](https://designsystem.digital.gov/utilities/layout-grid/) on outer most element of page (just inside `body` element). Later on, by seeing some examples like http://digital.gov/ and http://designsystem.digital.gov/ , I realized that [USWDS Layout grid](https://designsystem.digital.gov/utilities/layout-grid/) is not for outer most element of the page but rather for some sections inside the page. 

It would be nice if it can be elaborated more at [USWDS Layout grid](https://designsystem.digital.gov/utilities/layout-grid/) about how and where user can use [USWDS Layout grid](https://designsystem.digital.gov/utilities/layout-grid/) in whole page. Some examples would be great. I know there are some examples there but they are not in context with other non-layout-grid components.

Thank you.

**Comment by mejiaj** [comment]
Hey @gitcode7, there's no strict way to use the components. We provide guidance on best practices, but intentionally leave it to the implementor to decide how to best apply the design system to fit their needs.

You can see in the [landing page example](https://designsystem.digital.gov/templates/landing-page/) we use `grid-container` to contain some elements.

If you have any suggestions for improving the [container section in guidance](https://designsystem.digital.gov/utilities/layout-grid/#containers-rows-and-columns) please feel free to create an [issue](https://github.com/uswds/uswds-site/issues/new/choose).
